gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -610,13 +659,35 @@ gst_tensor_trainer_chain (GstPad * sinkpad, GstObject * parent,
           goto error;
         }
       }
-#if 0
-      /** @todo Need to updatd out_tensors */
-      /* get loss, accuracy, val_loss, val_accuracy */
-      double data[4] = { 0, 0, 0, 0 };
+
+      ret =
+          trainer->fw->getModelRunStats (trainer->fw, &trainer->prop,
+          trainer->privateData);
+      if (ret < 0) {
+        GST_ERROR_OBJECT (trainer,
+            "Faild to get stats from the model being trained in subplugin(%s).",
+            trainer->fw_name);
+        return GST_FLOW_ERROR;
+      }
+      /* If the value is invalid, it is already set by INVALID_VALUE. */
+      if (trainer->prop.training_loss > 0)
+        data[0] = trainer->prop.training_loss;
```

Comment:
Make a enum for ```training_loss```, ```training_accuracy```, ```validation_loss```, and ```validation_accuracy``` instead of using 0, 1, 2, 3.

Reply:
okay, thank you.